index.js

```diff
@@ -185,6 +186,10 @@ class Dynamodb extends Datastore {
             });
         }
 
+        if (config.sort) {
+            scanner = scanner[config.sort]();
```

Comment:
The API does the validation before it comes here, so we should assume that these options are valid. Same for paginate. We don't check if `config.paginate.page` is negative here because the API already does the validation work.